lib/presto/client/statement_client.rb

```diff
@@ -91,22 +90,30 @@ def debug?
       !!@options[:debug]
     end
 
-    def closed?
-      @closed
+    def running?
+      @state == :running
     end
 
-    attr_reader :exception
+    def client_aborted?
+      @state == :client_aborted
+    end
+
+    def client_error?
+      @state == :client_aborted
+    end
 
-    def exception?
-      @exception
+    def finished?
+      @state == :finished
     end
 
+    attr_reader :exception
```

Comment:
No one seems to set `exception` anymore. Is it still necessary?

Reply:
I don't think anyone use it